projects/Heavenly Jungle NFT

```diff
@@ -0,0 +1,53 @@
+## Heavenly Jungle NFT
+
+## Category 
+Application
+
+## Project Description
+DAO NFT Platform allowing fractional ownership. Includes minting, auctions, royalties, licensing and the option to rage quit.
+
+## Use of IPFS, Filecoin and Libp2p
+Currently using Pinata's IPFS pinning service. Looking for guidance on this matter.
+
+## Project Status
+Early Development
+
+## Previews
+<!--Add some screenshots to give a preview of your product-->
+
+## Target Audience
+Art Communities, Individual Artists
+
+## Rough estimated user base (if applicable)
+<!--How many users do you have right now?-->
+
+## Github repo
+<!--Attach a link to your GitHub repo - open source is required - please make sure your repo has a license file and is licensed using MIT open source license! -->
+https://github.com/Mol-LeArt/Heavenly-Jungle
+
+## Website
+<!--Link your website if available-->
+
+## Docs
+<!--Including a link to your project docs!-->
+
+## Team Info
+<!-- Introduce your amazing team - how many team members are working on this project and who are they?-->
+Daniel - front end dev
```

Comment:
Please link the Github accounts of team members

Reply:
Daniel - https://github.com/JFF-Danno
Stamford - https://github.com/audsssy